packages/ocs/dashboards/persistent-external/client-fallback.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * This hooks is for ODF Client Mode use case only.
```

Comment:
not a hook anymore :P
```suggestion
 * This is for ODF Client Mode use case only.
```